data/assets/scene/solarsystem/heliosphere/todayssun/grid.asset

```diff
@@ -0,0 +1,118 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms_heliosphere")
+local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
+
+
+
+local size = 700000000
+
+local carrington_prime_meridian = {
+  Identifier = "CarringtonPrimeMeridian",
+  Parent = sunTransforms.SunIAU.Identifier,
+  Transform = {
+    Translation = {
+      Type = "StaticTranslation",
+      Position = { 3500000, 0, 0 }
+    },
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0.0, 0.0, -math.pi / 2.0 } -- -90 degrees
+    },
+    Scale = {
+      Type = "StaticScale",
+      Scale = 698000000
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 1.0, 1.0, 0.0 },
+    LineWidth = 2.0,
+    LongSegments = 2,
+    LatSegments = 64
+  },
+  GUI = {
+    Name = "Carrington Prime Meridian",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[A 2x64 spherical grid showing the Carrington prime meridian]]
```

Comment:
```suggestion
    Description = [[A 2x64 spherical grid showing the Carrington prime meridian of the Sun. ]]
```
?

Do we need to add a sentence of what this is?  I would it a bit difficult to figure out by googling